Sources/Utils.swift

```diff
@@ -21,3 +27,42 @@ public func generateState(withLength len: Int) -> String {
     }
     return randomString
 }
+
```

Comment:
only one blank line, I think SwiftLint will warn about it

Reply:
Done